doc-changelog/action.yml

```diff
@@ -91,7 +99,49 @@ runs:
       run: |
         python -m pip install --upgrade pip towncrier==${{ inputs.towncrier-version }}
 
+    - name: "Get first letter of conventional commit type"
+      if: ${{ inputs.use-labels == 'false' }}
+      env:
+        PR_TITLE: ${{ github.event.pull_request.title }}
+      shell: python
+      run: |
+        import sys
+        sys.path.insert(1, '${{ github.action_path }}/../doc-changelog/')
+
+        from parse_pr_title import get_first_letter_case
+
+        pr_title = """${{ env.PR_TITLE }}"""
+        get_first_letter_case(pr_title)
```

Comment:
I did this instead of using a bash shell and running `python ${{ github.action_path }}/../doc-changelog/parse_pr_title.py` because of the PR_TITLE. I have to put triple quotes around the env.PR_TITLE in case the title contains double quotes, single quotes, and/or backticks

Reply:
Good job in thinking about quotes and backticks.

Wouldn't something like 
```python
pr_title = f"""{os.getenv("PR_TITLE")}"""
```
do the job though ? If I don't understand the problem, can you tell me what I'm missing here ?

Reply:
@SMoraisAnsys That would work too, but I think it would do the same thing as line 113. If env.PR_TITLE is 

docs: pr's title with "double quotes",   `backticks`, and ``double backticks``!     

Then I have to wrap it in triple quotes when I set it equal to the pr_title variable. 

In bash, I wasn't able to pass in the env.PR_TITLE as an argument to this command `python ${{ github.action_path }}/../doc-changelog/parse_pr_title.py --get-first-letter-case --pr_title ${{ env.PR_TITLE }}` because I couldn't find a way to wrap the env.PR_TITLE in the triple quotes equivalent for bash (if that's even a thing)

Reply:
so I think the issue down the line would be, I make the code above into a python script and run the command in bash, but then we run into the issue of how to pass the env.PR_TITLE value with special characters into the argument

Reply:
I think using multiple python files would solve this problem as well as the one with sys path setup. However, they will likely contain copy pasted line of codes and it will be a pain to maintain. This will change if we ever extract all of this into a single python package !